src/Components/People.js

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const People = () => {
+  const [input, setInput] = useState("");
+  const [people, setPeople] = useState([]);
+  const [hasSearched, setHasSearched] = useState(false);
+  const [person, setPerson] = useState([]);
+
+  const fetchPeople = async () => {
+    try {
+      const res = await axios.get(`https://ghibliapi.herokuapp.com/people`);
+      setPeople(res.data);
+    } catch (error) {
+      setPeople([]);
+    }
+  };
+
+
+  useEffect(() => {
+    fetchPeople();
+  }, [])
+
+  const handleInput = (e) => {
+    setInput(e.target.value);
+  };
+
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    people.forEach((person) => {
```

Comment:
ForEach works here, but I think either filter or find are easier methods for what you're going for